golang/demo.go

```diff
@@ -134,15 +134,13 @@ func httpRequest(method, urlHost string, reqParameters map[string]interface{}) (
 	}
 
 	urlParameters := url.Values{}
-	urlParameters.Add("access_id", ACCESSID)
-	urlParameters.Add("tonce", currentMilliseconds)
```

Comment:
I also think they should be removed from here according to their documentation, but why didn't you add them after making the signature?